proposals/utils/stepper.py

```diff
@@ -84,8 +87,17 @@ def build_stepper(
             raise RuntimeError(
                 "Base layout was never defined for this stepper",
             )
-        # First, insert all items into the layout
+        # First, insert all items into the layout & let them figure out their 
+        #own styling
         for item in self.items:
+            #Only check item.is_current until there is a current item found
+            while not self.current_item:
+                if item.is_current(self.request):
+                    self.current_item = item
+                    self.current_item_ancestor = item.get_ancestor()
+                else:
+                    break
+            item.get_css_classes()
```

Comment:
What is the reason that we need to get the CSS classes at this stage? Can't we just call `item.get_css_classes()` from the template?